source/features/remove-unused-repo-tabs.tsx

```diff
@@ -0,0 +1,111 @@
+import cache from 'webext-storage-cache';
+import select from 'select-dom';
+import elementReady from 'element-ready';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+import fetchDom from '../helpers/fetch-dom';
+import getTabCount from '../github-helpers/get-tab-count';
+import looseParseInt from '../helpers/loose-parse-int';
+import {getWorkflows} from './next-scheduled-github-action';
+import abbreviateNumber from '../helpers/abbreviate-number';
+import {getProjectsTab} from './remove-projects-tab';
+import {onlyShowInDropdown} from './more-dropdown';
+import {buildRepoURL, getRepo} from '../github-helpers';
+
+function tabCannotBeHidden(tab: HTMLElement | undefined): boolean {
+	if (
+		!tab || // Tab disabled 🎉
+		tab.matches('.selected') ||// User is on tab 👀
+		// Repo/Organization owners should see the tab. If they don't need it, they should disable the feature altogether
+		pageDetect.canUserEditRepo() ||
+		pageDetect.canUserEditOrganization()
+	) {
+		return true;
+	}
+
+	return false;
+}
+
+function setTabCounter(tab: HTMLElement, count: number): void {
+	const tabCounter = select('.Counter', tab)!;
+	tabCounter.textContent = abbreviateNumber(count);
+	tabCounter.title = count > 999 ? String(count) : '';
+}
+
+const getWikiPageCount = cache.function(async (): Promise<number> => {
+	const wikiPages = await fetchDom(buildRepoURL('wiki'), '#wiki-pages-box .Counter');
+	if (!wikiPages) {
+		return 0;
+	}
+
+	return looseParseInt(wikiPages);
```

Comment:
If it doesn't already, `looseParseInt` should just return `0` when the input is undefined, so you can drop the condition above as well
```suggestion
	return looseParseInt(wikiPages);
```